dockerfiles/Dockerfile.router

```diff
@@ -1,39 +1,43 @@
 ARG DEBUG_IMAGE
 # Build is required to extract the release files
-FROM --platform=linux/amd64 alpine:latest AS build
+FROM --platform=linux/amd64 debian:bullseye-slim AS build
```

Comment:
As noted in the PR body, this accounts for the fact that the installer necessitates an architecture that we can actually run the Router on and switches `alpine:latest` for a specific version of `debian:*slim` images.

There is no material change on the outcome of the image — we were using the Alpine image to download a tarball for the next build stage, **not** to create a small output image, **nor** on account of producing a Router image that runs on Alpine.  Today, the Router does not support running on Alpine for reasons we will solve in the future.